var/spack/repos/builtin/packages/py-pyopengl/package.py

```diff
@@ -0,0 +1,88 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import sys
+
+from spack.package import *
+
+
+class PyPyopengl(PythonPackage):
+    """PyOpenGL is the most common cross platform Python binding to OpenGL and related APIs."""
+
+    homepage = "http://pyopengl.sourceforge.net"
+    url = "https://files.pythonhosted.org/packages/py3/p/pyopengl/PyOpenGL-3.1.6-py3-none-any.whl"
+    list_url = "https://pypi.org/simple/pyopengl/"
+
+    version(
+        "3.1.6",
+        sha256="a7139bc3e15d656feae1f7e3ef68c799941ed43fadc78177a23db7e946c20738",
+        expand=False,
+    )
+
+    variant(
+        "gl",
+        default="glx" if sys.platform.startswith("linux") else "other",
+        values=("glx", "osmesa", "other"),
+        multi=False,
+        description="The OpenGL provider to use",
+    )
+    variant("glu", default=True)
+    variant("glut", default=True)
+
+    conflicts("osmesa", when="gl=glx")
+    conflicts("osmesa", when="gl=other")
+    conflicts("glx", when="gl=osmesa")
+    conflicts("glx", when="gl=other")
+
+    depends_on("python@3.6:", type=("build", "run"))
```

Comment:
Spack only supports 3.7+ anyway, you can remove this line

Reply:
Done.